dbt/adapters/databricks/api_client.py

```diff
@@ -0,0 +1,85 @@
+from requests.exceptions import HTTPError
+from typing import Any, Dict
+
+from databricks_cli.sdk.api_client import ApiClient
+
+
+class Api12Client:
+    def __init__(self, host: str, token: str, command_name: str = ""):
+        self._api_client = ApiClient(
+            user="token",
+            password=token,
+            host=f"https://{host}",
+            api_version="1.2",
+            command_name=command_name,
```

Comment:
Is this `command_name` being recorded somewhere after the run?

Reply:
It will be part of user agent in the API request.

Reply:
@ChenyuLInx @jtcohen6 btw, is it possible to set a simple string in the user agent of `dbt-spark` to identify the API calls are from `dbt` similar to ODBC connections?

Reply:
@jtcohen6  I am not very familiar with what we did with ODBC connections, so I will defer it to you.
@ueshin feel free to open an issue in dbt-spark about this!

Reply:
@ChenyuLInx I opened an issue in dbt-spark https://github.com/dbt-labs/dbt-spark/issues/460. Thanks.